app/Jobs/CloneImagesOrVideos.php

```diff
@@ -17,6 +17,7 @@
 use Biigle\VideoAnnotationLabel;
 use Biigle\VideoLabel;
 use Biigle\Volume;
+use Illuminate\Database\Eloquent\Collection;
```

Comment:
I think it's rather unusual to use a database collection directly. Usually, this is only returned from a database query. Instead, if you want to use a collection, just use the `collect([])` helper. This gives you a more generic `Illuminate\Support\Collection`.